assets/css/admin.scss

```diff
@@ -3512,13 +3512,32 @@
 		 input[type="text"],
 		 input[type="email"],
 		 input[type="number"],
+		 input[type="password"],
+		 input[type="datetime"],
+		 input[type="datetime-local"],
+		 input[type="date"],
+		 input[type="time"],
+		 input[type="week"],
+		 input[type="url"],
+		 input[type="tel"],
 		 input.regular-input {
 			 width: 400px;
 			 margin: 0;
 			 padding: 6px;
 			 box-sizing: border-box;
 			 vertical-align: top;
 		 }
+		 
+		 input[type="datetime-local"],
+		 input[type="date"],
+		 input[type="week"],
+		 input[type="tel"] {
+			 width: 200px;
+		 }
+		 
+		 input[type="time"] {
+			 width: 100px;
```

Comment:
100px doesn't seem quite big enough. In testing, the "M" in "AM" was cut off:
<img width="116" alt="screen shot 2018-08-13 at 11 38 50 am" src="https://user-images.githubusercontent.com/7317227/44051051-a5763c40-9eed-11e8-938b-519080dc5898.png">
<img width="156" alt="screen shot 2018-08-13 at 11 41 21 am" src="https://user-images.githubusercontent.com/7317227/44051121-d521be92-9eed-11e8-8d50-6c81339eec4a.png">

My recommendation is to make this input 200px wide like the other date inputs.

Also, to regenerate the CSS files there is a grunt task: `grunt css`. It's not necessary for this PR since all the assets will get recompiled before the release, but it may help in your testing.

Reply:
I fixed it in my branch, I think it updates the pull request, right?

Reply:
Yep the pull request is synced to the branch